modules/ldap-auto-syncing.adoc

```diff
@@ -209,9 +210,10 @@ spec:
 ----
 <1> Configure the settings for the cron job. See "Creating cron jobs" for more information on cron job settings.
 <2> The schedule for the job specified in link:https://en.wikipedia.org/wiki/Cron[cron format]. This example cron job runs every 30 minutes. Adjust the frequency as necessary, making sure to take into account how long the sync takes to run.
-<3> The LDAP sync command for the cron job to run. Passes in the sync configuration file that was defined in the config map.
-<4> This secret was created when the LDAP IDP was configured.
-<5> This config map was created when the LDAP IDP was configured.
+<3> How long to keep finished jobs. This should match the period of the job schedule to clean old failed jobs and prevent unnecessary alerts. See link:https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished[Kubernetes - TTL-after-finished Controller].
```

Comment:
```suggestion
<3> How long, in seconds, to keep finished jobs. This should match the period of the job schedule in order to clean old failed jobs and prevent unnecessary alerts. For more information, see link:https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished[TTL-after-finished Controller] in the Kubernetes documentation.
```

Just a few minor wording suggestions if you wouldn't mind updating. (Please make the changes manually and then squash - don't use the "Commit suggestion" button shown in the GitHub UI here)

Reply:
Thank you for the suggestion, text tweaked